src/pushsource/_impl/helpers.py

```diff
@@ -85,3 +93,22 @@ def try_bool(value):
     elif value in ["", "0", "false", "no"]:
         return False
     raise ValueError("Expected a boolean, got %s" % repr(value))
+
+
+def force_https(url: str) -> str:
+    """Force ``url`` to use https as it's scheme.
```

Comment:
```suggestion
    """Force ``url`` to use https as its scheme.
```

("it's" always means "it is" or "it has")